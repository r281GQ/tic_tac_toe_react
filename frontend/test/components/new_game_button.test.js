import React from 'react';
import { shallow } from 'enzyme';
import NewGameButton from './../../src/components/new_game_button';

describe('NewGameButton test', () => {
  it('should render', () => {
    const component = shallow(
      <NewGameButton createNewGameHandler={value => value} />
    );
    expect(component).toHaveLength(1);
  });
});
