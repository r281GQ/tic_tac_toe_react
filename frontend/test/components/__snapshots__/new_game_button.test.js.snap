// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewGameButton test should render 1`] = `
<IconButton
  disableTouchRipple={false}
  disabled={false}
  iconStyle={Object {}}
  onClick={[Function]}
  tooltip="New game"
  tooltipPosition="bottom-center"
  touch={false}
>
  <FontIcon
    className="fa fa-plus-circle white"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  />
</IconButton>
`;
